beacon/beacon-readablestream.window.js

```diff
@@ -0,0 +1,23 @@
+function assert_beacon(stream) {
```

Comment:
Beacon with streams is never going to work because it's keepalive + ReadableStream which is rejected by another reason.